plugins/opsgenie/plugin.spec.yaml

```diff
@@ -0,0 +1,277 @@
+      visibleTo:
+        title: VisibleTo
+        description: Teams and users that the alert will become visible to without sending any notification. Type field is mandatory for each item, where possible values are team and user. In addition to the type field, either ID or name should be given for teams and either ID or username should be given for users. Please note that alert will be visible to the teams that are specified within responders field by default, so there is no need to re-specify them within visibleTo field. You can refer below for example values
+        type: "[]object"
+        required: false
+        example: '[{"id":"4513b7ea-3b91-438f-b7e4-e3e54af9147c","type":"team"},{"name":"example_name","type":"team"}]'
+      actions:
+        title: Actions
+        description: Custom actions that will be available for the alert
+        type: "[]string"
+        required: false
+        example: '["Restart", "AnExampleAction"]'
+      tags:
+        title: Tags
+        description: Tags of the alert
+        type: "[]string"
+        required: false
+        example: '["OverwriteQuietHours","Critical"]'
+      details:
+        title: Details
+        description: Map of key-value pairs to use as custom properties of the alert
+        type: object
+        required: false
+        example: '{"key1":"value1","key2":"value2"}'
+      entity:
+        title: Entity
+        description: Entity field of the alert that is generally used to specify which domain an alert is related to
+        type: string
+        required: false
+        example: "An example entity"
+      source:
+        title: Source
+        description: Source field of the alert. Default value is IP address of the incoming request
+        type: string
+        required: false
+        example: "192.168.0.1"
+      priority:
+        title: Priority
+        description: Priority level of the alert. Possible values are P1, P2, P3, P4 and P5. Default value is P3
+        type: string
+        enum:
+          - ""
+          - "P1"
+          - "P2"
+          - "P3"
+          - "P4"
+          - "P5"
+        default: "P3"
+        required: false
+        example: "P1"
+      user:
+        title: User
+        description: Display name of the request owner
+        type: string
+        required: false
+        example: "ExampleName"
+      note:
+        title: Note
+        description: Additional note that will be added while creating the alert
+        type: string
+        required: false
+        example: "Example additional note"
+    output:
+      result:
+        title: Result
+        description: Result message from API
+        type: string
+        required: false
+      took:
+        title: Took
+        description: Time took to execute API
+        type: float
+        required: false
+      requestId:
+        title: Request ID
+        description: ID of a executed API request
+        type: string
+        required: false
```

Comment:
Is it really possible that these outputs may not come back in a success scenario? If they always come back when the action is successful, then these should all be changed to `required: true`.